suite-native/module-dev-utils/package.json

```diff
@@ -15,6 +15,7 @@
         "@react-navigation/stack": "^6.2.2",
         "@suite-native/atoms": "*",
         "@suite-native/config": "*",
+        "@suite-native/module-settings": "*",
```

Comment:
We should not have dependency between modules. We can probably do exception here because it's purely dev module but keep that in mind.